lib/openhab/core/value_cache.rb

```diff
@@ -0,0 +1,188 @@
+# frozen_string_literal: true
+
+OpenHAB::Core.import_preset("cache")
+unless defined?($sharedCache)
+  $sharedCache = $privateCache = nil
```

Comment:
Since you've removed private_cache, should $privateCache be removed here (and a few other places in this PR) too?

Reply:
Yup. I thought I had already removed this line. Will do.